docs/sender/data.md

```diff
@@ -253,3 +254,35 @@ config = {"address": "devo.collertor", "port": 443,
 con = Sender.for_logging(config=config, tag="my.app.test.logging")
 logger = get_log(name="devo_logger", handler=con)
 ```
+## Enabling verification for SenderConfigSSL configuration file
+
+To help troubleshoot any problems with the configuration file the variables:
+
++ address **(_tuple_)**: (Server address, port)
++ key **(_str_)**: key src file
++ cert **(_str_)**: cert src file
++ chain **(_str_)**: chain src file
+
+Can be verify by adding "verify_config": true to the configuration file, in case any of the variables is invalid or incompatible with each other a DevoSenderException will be raised indicating the variable that’s causing the trouble, below an example of the file and an exception:
```

Comment:
**nitpick:**
1. If the sentence is passive, the verb should be in past: "Can be verified". 
2. Could you wrap `"verify_config": true` between backticks to distinguish code from plain text, please? The same goes for `DevoSenderException`.